src/modules/UI/Settings/selectors.js

```diff
@@ -59,6 +59,15 @@ export const getDisplayDenominationFromSettings = (settings: any, currencyCode:
   return selectedDenomination
 }
 
+export const getDisplayDenominationFull = (state: State, currencyCode: string) => {
```

Comment:
What is the intended difference between this routine and getDisplayDenomination?

Seems that the only diff is that getDisplayDenomination will return fiat denominations if given a fiat currencyCode.

Reply:
Solely for debugging purposes. Having the `getDisplayDenomination` function call another function, which calls another function, etc makes it more of a hassle to debug with breakpoints.